mlflow/projects/databricks.py

```diff
@@ -295,24 +246,36 @@ class DatabricksSubmittedRun(SubmittedRun):
     project. Note that run_id may be None, e.g. if we did not launch the run against a tracking
     server accessible to the local client.
     """
-    def __init__(self, databricks_run_id, run_id):
+    def __init__(self, databricks_run_id, run_id, profile):
```

Comment:
What do you think about this one taking a DatabricksJobRunner instead of a profile?

Reply:
Yep, that seems reasonable - then conceptually `DatabricksJobRunner` contains all the API-request logic needed to launch & manage a project run, which seems nice to reason about.